Makefile

```diff
@@ -2,7 +2,7 @@
 
 app_name=talk_matterbridge
 
-matterbridge_version=1.23.2
+matterbridge_version=1.25.2
```

Comment:
```suggestion
matterbridge_version=1.24.2
```
?

Reply:
1.25 is available so may as well upgrade to that: https://github.com/42wim/matterbridge/releases/tag/v1.25.2

Reply:
Ah right, it's not the nextcloud version :D
It's been too long